test/test_all.py

```diff
@@ -703,15 +714,15 @@ class ExampleJitterNoPython(ExampleJitter):
             "'py-llvm 3.2' module is required for llvm tests"
 
         # Remove llvm tests
-        if TAGS["llvm"] not in exclude_tags:
+        if (not include_tags and TAGS["llvm"] not in exclude_tags):
```

Comment:
Oh! I don't know: I was not sure about me on the review of this case.
I may prefer the abort case, but it's up to you.
If you are aware of the case and prefer to let the code like this, tell me.